cmd/meroxa/root/apps/apps.go

```diff
@@ -24,53 +24,76 @@ import (
 
-	"github.com/meroxa/cli/log"
 	"github.com/meroxa/cli/utils/display"
 	"github.com/spf13/cobra"
 )
 
 type ApplicationState string
 
 const (
-	ApplicationStateInitialized ApplicationState = "initialized"
-	ApplicationStateDeploying   ApplicationState = "deploying"
-	ApplicationStatePending     ApplicationState = "pending"
-	ApplicationStateRunning     ApplicationState = "running"
-	ApplicationStateDegraded    ApplicationState = "degraded"
-	ApplicationStateFailed      ApplicationState = "failed"
-
-	collectionName = "apps"
+	AppProvisioningState   ApplicationState = "provisioning"
+	AppProvisionedState    ApplicationState = "provisioned"
+	AppDeprovisioningState ApplicationState = "deprovisioning"
+	AppDeprovisionedState  ApplicationState = "deprovisioned"
+	AppDegradedState       ApplicationState = "degraded"
+
+	deploymentCollection  = "conduit_deployments"
+	applicationCollection = "conduit_apps"
 )
 
 var displayDetails = display.Details{
-	"Name":        "name",
-	"State":       "state",
-	"SpecVersion": "specVersion",
-	"Created":     "created",
-	"Updated":     "updated",
+	"Name":              "name",
+	"State":             "state",
+	"ApplicationSpec":   "stream_tech",
+	"Config":            "config",
+	"PipelineFilenames": "pipelines_filenames",
+	// "PipelineEnriched":  "pipeline_enriched",
+	// "PipelineOriginal":  "pipeline_original",
+	"Created": "created",
+	"Updated": "updated",
+}
+
+type Deployment struct {
+	ID              string `json:"id,omitempty"`
+	Archive         string `json:"archive"`
+	State           string `json:"state,omitempty"`
+	ApplicationSpec string `json:"app_spec,omitempty"`
+
+	Created            AppTime `json:"created,omitempty"`
+	Updated            AppTime `json:"updated,omitempty"`
+	ProcessorPlugins   string  `json:"processors_plugins,omitempty"`
+	ProcessorFilenames string  `json:"processors_filenames,omitempty"`
+	PipelineFilenames  string  `json:"pipeline_filenames,omitempty"`
+}
+
+type Deployments struct {
+	Page       int          `json:"page"`
+	PerPage    int          `json:"perPage"`
+	TotalItems int          `json:"totalItems"`
+	TotalPages int          `json:"totalPages"`
+	Items      []Deployment `json:"items"`
 }
 
 // Application represents the Meroxa Application type within the Meroxa API.
 type Application struct {
-	ID          string                 `json:"id"`
-	Name        string                 `json:"name"`
-	State       ApplicationState       `json:"state"`
-	Spec        map[string]interface{} `json:"spec"`
-	SpecVersion string                 `json:"specVersion"`
-	Created     AppTime                `json:"created"`
-	Updated     AppTime                `json:"updated"`
-	Image       string                 `json:"imageArchive"`
+	ID                string   `json:"id"`
+	DeploymentID      []string `json:"deployment_id"`
+	Name              string   `json:"name"`
+	State             string   `json:"state"`
+	ApplicationSpec   string   `json:"stream_tech"`
```

Comment:
this was renamed to `stream_provider` not sure where `stream_tech` is coming from?